javalib/src/main/scala/java/util/logging/LogRecord.scala

```diff
@@ -0,0 +1,83 @@
+package java.util.logging
+
+import java.util.concurrent.atomic.AtomicLong
+
+object LogRecord {
+  protected[logging] var sequence:AtomicLong = new AtomicLong(0)
+}
+
+class LogRecord protected[logging] (private[this] var level: Level,
+    private[this] var msg: String,
+    private[this] var sourceClassName: Option[String] = None,
+    private[this] var sourceMethodName: Option[String] = None,
+    private[this] var params: List[AnyRef] = Nil,
+    private[this] var thrown: Option[Throwable] = None,
+    private[this] var loggerName: Option[String] = None) {
```

Comment:
you should not define a second constructor on this class. Put all the `var` definitions in the class and use `(level: Level, msg: String)` as the only constructor.